docs/MANUAL.md

```diff
@@ -0,0 +1,215 @@
+
+```ini
+# By default, a piece of code is considered duplicated as soon as there are at least
+# 100 successive and duplicated tokens spread over at least 10 lines of code.
+sonar.cpd.delphi.minimumTokens=200
+sonar.cpd.delphi.minimumLines=20
+```
+
+For more details on SonarQube configuration options, see
+[the SonarQube documentation](https://docs.sonarsource.com/sonarcloud/advanced-setup/analysis-parameters/).
+
+In addition to `sonar.tests`, SonarDelphi allows you to configure certain types and their methods as test code.
+This is designed for compatibility with common Delphi test frameworks and can be configured in two ways:
+
+```ini
+# Code within this type and all descendants is considered test code.
+# Defaults to DUnit's test base type: TestFramework.TTestCase.
+sonar.delphi.testType=TestFramework.TTestCase
+# Code within any type annotated with this attribute is considered test code.
+# Defaults to DUnitX's test attribute: DUnitX.Attributes.TestFixtureAttribute.
+sonar.delphi.testAttribute=DUnitX.Attributes.TestFixtureAttribute
+```
+
+For the full list of SonarDelphi configuration options, see [here](CONFIGURATION.md).
+
+## Run a SonarDelphi scan
+
+Once your project is configured, you simply
+[run the scanner](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/scanners/sonarscanner/)
+in the same directory as your `sonar-project.properties`.
+
+After an analysis, the project page on SonarQube will refresh with updated issues and metrics.
+Browsing the issues, you have the ability to update them if needed - assign them to other users, add comments,
+or change their status (e.g. accepting issues you won't fix or hiding false positives).
+
+If there are a large number of issues for rules that don't suit your codebase - for example, issues complaining
+your classes don't begin with `T` when you have a different convention - you may benefit from
+[customizing SonarDelphi's analysis rules](#configure-analysis-rules).
+
+## Configure analysis rules
+
+The rules applied during a scan are dictated by the project's
+[quality profile](https://docs.sonarsource.com/sonarqube/latest/instance-administration/quality-profiles/).
+
+For Delphi, the default quality profile ("Sonar way") contains sensible default rules that are generally
+accepted to be best practice. These may not be helpful to apply to your codebase if it has its own conventions that
+conflict with the rules. Some common quality profile changes include:
+
+* Setting `excludeApi` to `true` (allowing "unused" public API) on `UnusedConstant`, `UnusedField`,
+  `UnusedGlobalVariable`, `UnusedProperty`, `UnusedRoutine`, and `UnusedType`
+* Changing the `prefixes` field on `ClassName`, `RecordName`, `UnitName`, `EnumName`, `InterfaceName`,
+  and `HelperName`
+* Disabling `ConstantName`
+* Disabling `ShortIdentifier`
+* Disabling `NilComparison`
+
+While SonarDelphi's quality profile contains many useful rules, there are also many non-default rules
```

Comment:
It would be nice to make this part a bit more terse. At minimum, I think we can lose the mention of the default quality profile.

It could be as simple as:
```md
Non-default rules that are often added to quality profiles include:
```

Reply:
How about:

> SonarDelphi also provides many non-default rules that are more niche or opinionated.
> Some rules that are often added to quality profiles include:

Reply:
Sure, let's go with that.